### PR TITLE
DEVDOCS-4605: [update] Payments, add bigpay_token

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4605]

## What changed?
Added bigpay_token

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4605]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -2043,9 +2043,51 @@ components:
...
                   example: 35.42
-      title: Transaction
-      x-internal: false
       type: object
+      title: ''
+      x-examples:
+        Example 1:
+          event: purchase
+          method: credit_card
+          amount: -3.402823669209385e+38
```
Comment (bc-tgomez) [inline reference/orders.v3.yml:2052]: Thinking that this here needs to be adjusted.

**Review by bc-tgomez — COMMENTED** [review]
Just want to take a look at that one comment but other than that, I think we're good to go. 😄

File: reference/orders.v3.yml (hunk 1 above)
Comment (bc-traciporter) [reply reference/orders.v3.yml]: Good find @bc-tgomez. I made the number positive.

**Review by slsriehl — CHANGES_REQUESTED** [review]
Let's wait on @bc-keith to make sure we're synced up with Payments. Presumably there will also be changes to at least a few of payment_processing, payment_methods, and customers.v3 list stored instruments.

**Review by slsriehl — APPROVED** [review]
Looks good. Nice additional changes.

**Pull request merged**